cmd/vela-slack/main.go

```diff
@@ -377,3 +417,71 @@ func run(c *cli.Context) error {
 	// execute the plugin
 	return p.Exec()
 }
+
+// Retrieves sAMAccountName from LDAP server using build author's email.
+func getLanID(c *cli.Context) string {
```

Comment:
@kaymckay what are your thoughts on changing the name to `getSAMAccountName` to match the functionality?